js/screens/Login/LoginScreen.js

```diff
@@ -42,27 +41,22 @@ class LoginContainer extends Component {
   handleSubmit() {
```

Comment:
Your _signInAsync function is not storing actual user token, plus it is not being called.